R/exclamation.R

```diff
@@ -40,6 +40,7 @@ exclamation <- c(
   "yahoo",
   "yay",
   "yeah",
+  "yeahyeah"
```

Comment:
Thanks! Did you mean to add yeahyah?

Btw. can you please also add a comma at the end? 

Thanks!